sopel_modules/twitter/twitter.py

```diff
@@ -1,28 +1,43 @@
 # coding=utf-8
-from __future__ import unicode_literals, absolute_import, division, print_function
+from __future__ import annotations, unicode_literals, absolute_import, division, print_function
```

Comment:
`__future__.annotations` doesn't exist [prior to Python 3.7](https://docs.python.org/3/library/__future__.html). Sopel 7.x still supports older versions, and therefore this import cannot be present in a plugin intended for use with Sopel 7.

I'm not even talking about py2.7, really. This will break things on 3.6, too.

```
$ python3.6 -i
Python 3.6.9 (default, Oct  8 2020, 12:12:24)
[GCC 8.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from __future__ import annotations
  File "<stdin>", line 1
SyntaxError: future feature annotations is not defined
```

Reply:
Augh